packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -1345,6 +1343,28 @@ class CrudStoreImpl
       countOptions,
     });
 
+    // Only check if index was used if query filter or sort is not empty
+    if (!isEmpty(query.filter) || !isEmpty(query.sort)) {
```

Comment:
We’re not doing a time limit for the original evaluation of the query here anymore?

Reply:
No, discussed with Julia and based on JM feedback we are removing the threshold (at least for the MVP)